docs/user_guide/2_software_description.md

```diff
@@ -1042,3 +1042,166 @@ Next, go to the 'Stored States' tab in 'Motion Planning'. Here you have full con
 .. image:: ../img/save_state.png
 ```
 
+## Creating a new world/scene
+
```

Comment:
Might be worth mentioning the fact that there is a GUI here. If I didn't like command-line interfaces, I might be a little frustrated to follow a full command-line tutorial only to find out that there is a GUI once I'd completed it.

Reply:
Done